az-core/src/main/java/azkaban/metrics/MetricsManager.java

```diff
@@ -114,27 +115,27 @@ public Timer addTimer(final String name) {
    * reporting metrics to remote metrics collector. Note: this method must be synchronized, since
    * both web server and executor will call it during initialization.
    */
-  public synchronized void startReporting(final String reporterName, final Props props) {
+  public synchronized void startReporting(final Props props) {
     final String metricsReporterClassName = props.get(CUSTOM_METRICS_REPORTER_CLASS_NAME);
-    final String metricsServerURL = props.get(METRICS_SERVER_URL);
-    if (metricsReporterClassName != null && metricsServerURL != null) {
+    final String metricsReporterConfigPath =
```

Comment:
Can you please move this to inner if case? It is not needed outside.

Reply:
done, thanks!